gdal/ogr/ogrsf_frmts/dxf/drv_dxf.html

```diff
@@ -72,16 +81,20 @@
 as described here.
 
 <ul>
-<li> A new layer will be available called blocks.  It will contain one or
+<li> A new layer will be available called "blocks".  It will contain one or
 more features for each BLOCK defined in the file.  In addition to the usual
-attributes, they will also have a BlockName attribute indicate what block
-they are part of.
+attributes, they will also have a Block attribute indicating what block
+they are part of. (Note, in GDAL 2.2.2 and earlier this attribute was called
+BlockName.)
```

Comment:
"GDAL &lt; 2.3" 
(there will likely be a GDAL 2.2.3 without those trunk-only changes)